src/styles/block/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &--4 &__star:nth-child(-n + 4) {
```

Comment:
apply styles for all modifier cases (1-5 stars)

Reply:
still not fixed, you need to copy all styles from stars task
<img width="636" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/709a5afb-4dc1-4bdf-acab-6a4ee8b3ef76">